test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -94,7 +95,7 @@ internal void CreateDotnetToolProject(string solutionRoot, string projectName, s
                 <!-- Things that do change-->
                 <RestorePackagesPath>{restorePackagesPath}</RestorePackagesPath>
                 <RestoreSolutionDirectory>{restoreSolutionDirectory}</RestoreSolutionDirectory>
-    
+                <MSBuildProjectExtensionsPath>{msbuildProjectExtensionsPath}</MSBuildProjectExtensionsPath>
```

Comment:
It's not valid to set this property here, as it's already been used to import .props from the Microsoft.Common.props.  MSBuild will now generate an error for this (see https://github.com/Microsoft/msbuild/pull/3059), so I would expect test errors once that change makes it to the MSBuild that you use.

The simplest fix is probably to remove `Sdk="Microsoft.NET.Sdk"` from the `Project` tag], and have a `PropertyGroup` that sets this property before an explicit Sdk.props import.  You'll also need an explicit Sdk.targets import at the end of the project file.

Reply:
@dsplaisted 
This test is for dotnet tool...the project style is DotnetToolReference...not packagereference, so there will be no error. 

This is the non-build scenario that we didn't want to break.

Reply:
@nkolev92 Changing the `MSBuildProjectExtensionsPath` after `Microsoft.Common.props` has been evaluated will be an error regardless of whether a project is using NuGet, or what the NuGet reference style is.

here will be a warning which NuGet enables if the `BaseIntermediateOutputPath` doesn't match the `MSBuildProjectExtensionsPath` which wouldn't apply here, but the error will still apply.

Reply:
Ahh, true. 

This will cause a failure, when we start using an updated version of CLI for our tests. 
I've created a tracking issue. 

https://github.com/NuGet/Home/issues/6769

@dsplaisted 
When is that change going into CLI?
I think there'll be a change in the template needed in the DotnetTool implementation /cc @wli3